src/subcommand/server.rs

```diff
@@ -6206,6 +6245,56 @@ next
     assert_eq!(children_json.page, 1);
   }
 
+  #[test]
+  fn children_recursive_indexed_endpoint() {
+    let server = TestServer::builder().chain(Chain::Regtest).build();
+    server.mine_blocks(1);
+
+    let parent_txid = server.core.broadcast_tx(TransactionTemplate {
+      inputs: &[(1, 0, 0, inscription("text/plain", "hello").to_witness())],
+      ..default()
+    });
+
+    let parent_inscription_id = InscriptionId {
+      txid: parent_txid,
+      index: 0,
+    };
+
+    server.assert_response(
+      format!("/r/children/{parent_inscription_id}/inscriptions/at/-1"),
+      StatusCode::NOT_FOUND,
+      &format!("inscription {parent_inscription_id} not found"),
+    );
+
+    server.mine_blocks(1);
+
+    let mut builder = script::Builder::new();
+    for _ in 0..70 {
+      builder = Inscription {
+        content_type: Some("text/plain".into()),
+        body: Some("hello".into()),
+        parents: vec![parent_inscription_id.value()],
+        unrecognized_even_field: false,
+        ..default()
+      }
+      .append_reveal_script_to_builder(builder);
+    }
+
+    let witness = Witness::from_slice(&[builder.into_bytes(), Vec::new()]);
+
+    let txid = server.core.broadcast_tx(TransactionTemplate {
+      inputs: &[(2, 0, 0, witness), (2, 1, 0, Default::default())],
+      ..default()
+    });
+
+    server.mine_blocks(1);
+    let latest_child_inscription_id = InscriptionId { txid, index: 69 };
+    let child_json = server.get_json::<api::ChildInscriptionRecursive>(format!(
+      "/r/children/{parent_inscription_id}/inscriptions/at/-1"
+    ));
+    assert_eq!(child_json.id, latest_child_inscription_id);
+  }
+
```

Comment:
Need test for `/at/0` which would be the first child inscription. As well as `/at/1` which is the second child. This checks the off by one error we had in the original sat endpoint PR see discussion about that [here](https://github.com/ordinals/ord/pull/2452#issuecomment-1791815670)

Look at the sat endpoint for reference of `/at/n` tests:

Reply:
will do.

Reply:
done.